pkgs/development/python-modules/pyxnat/default.nix

```diff
@@ -10,7 +10,7 @@
 buildPythonPackage rec {
   pname = "pyxnat";
   version = "1.4";
-  disabled = isPy27;
+  disabled = ! (pythonAtLeast "3.7");
```

Comment:
```suggestion
  disabled = pythonOlder "3.8";
```
Why though?

Reply:
I did it because that was the version where pathlib was added to the python stdlib.

Reply:
Then pathlib should be added for Python 3.6.

Reply:
Sure. I didn't realize we were maintaining nixpkgs for python3.6.

Reply:
We still have `python36` but I guess most packages are not tested on it.